ivy_tests/test_ivy/test_frontends/test_torch/test_utilities.py

```diff
@@ -0,0 +1,42 @@
+# global
+from hypothesis import strategies as st
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+
+@handle_frontend_test(
+    fn_tree="torch.result_type",
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("valid"),
+        valid_axis=True,
+        max_axes_size=1,
+    ),
+    keepdims=st.booleans(),
+)
+def test_torch_result_type(
+    dtype_x_axis,
+    keepdims,
+    as_variable,
+    with_out,
+    num_positional_args,
+    native_array,
+    on_device,
+    fn_tree,
+    frontend,
+):
+    input_dtype, x, axis = dtype_x_axis
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=with_out,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        frontend=frontend,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        input=x[0],
+        axis=axis,
+        keepdims=keepdims,
```

Comment:
these are not the args of this function so they shouldn't really be here

Reply:
Should I remove input=x(0) also?

Reply:
@guillesanbri do i also need remove input=x[0].